clients/spark/core/src/main/scala/io/treeverse/gc/UncommittedGarbageCollector.scala

```diff
@@ -221,7 +221,9 @@ object UncommittedGarbageCollector {
     val reportDst = reportPath(storageNamespace, runID)
     writeJsonSummary(reportDst, runID, firstSlice, startTime, success, expiredAddresses.count())
 
-    expiredAddresses.write.parquet(s"${reportDst}/deleted")
+    val cachedAddresses = expiredAddresses.cache()
```

Comment:
Can we add a comment or TODO here, that we need to remove the caching + text file once we have a working backup solution?

Reply:
Will open an issue - thanks!

Reply:
https://github.com/treeverse/lakeFS/issues/4794